assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -161,31 +161,33 @@ export const actions = {
 		invariant( profileName, 'profileName is required.' );
 		invariant( timezone, 'timezone is required.' );
 
-		let accountTicket;
+		let response,
+			error;
 		yield {
 			payload: {},
 			type: START_FETCH_CREATE_ACCOUNT,
 		};
 
 		try {
-			accountTicket = yield {
+			response = yield {
 				payload: { accountName, propertyName, profileName, timezone },
 				type: FETCH_CREATE_ACCOUNT,
 			};
-
+			const { accountTicket } = response;
 			yield actions.receiveCreateAccount( accountTicket );
 
 			yield {
 				payload: {},
 				type: FINISH_FETCH_CREATE_ACCOUNT,
 			};
-		} catch ( error ) {
-			return {
+		} catch ( e ) {
+			error = e;
+			yield {
 				payload: { error },
```

Comment:
The payload here should be the same in for all actions, except for the `CATCH` case which has the added error.

Reply:
So include `{ accountName, propertyName, profileName, timezone }` as payload for each yielded action? Will do.